django_netjsonconfig/base/template.py

```diff
@@ -53,6 +64,43 @@ class AbstractTemplate(BaseConfig):
                                                 'be automatically managed behind the scenes '
                                                 'for each configuration using this template, '
                                                 'valid only for the VPN type'))
+    sharing = models.CharField(_('Sharing settings'),
+                               choices=SHARING_CHOICES,
+                               default='private',
+                               max_length=16,
+                               db_index=True,
+                               help_text=_('Whether to keep this template private, share it publicly, '
+                                           'share it privately using a secret key or import its '
+                                           'contents from another source'))
+    key = models.CharField(max_length=64,
+                           null=True,
+                           blank=True,
+                           default=get_random_key,
+                           validators=[key_validator],
+                           verbose_name=_('Secret key'),
+                           help_text=_('secret key needed to import the template'))
```

Comment:
the first letter of this line is lowercase while all other `help_text` have the first letter uppercase